UpdateService/handler.ts

```diff
@@ -173,8 +188,11 @@ export function UpdateServiceHandler(
                 userInfo.token_name
               )
             )
-            .chain(service =>
-              getSubscriptionKeysTask(
+            .chain(service => {
```

Comment:
Can we flat this chain-pipeline? In theory, the following
```ts
.chain(
  foo1.chain(
    foo2.chain(
       foo3
    )
  )
)
```

It's equal to 
```ts
.chain( foo1 )
.chain( foo2 )
.chain( foo3 )

Reply:
In order to flat chain-pipeline we have to build a struct that contains a subset of right values for each chain, due to usage of this values in the last chain ex:
```ts
.chain(() =>
  foo1.chain(bar1 =>
    foo2.chain(bar2 =>
       foo3(bar1, bar2)
    )
  )
)
```